apps/shipments/models/shipment.py

```diff
@@ -511,6 +523,24 @@ def drop_off(self, document_id=None, raw_asset_physical_id=None, **kwargs):
 
         self.delivery_act = datetime.now(timezone.utc)  # TODO: pull from action parameters?
 
+    def validate_gtx(self):
+        validation_status = GTXValidation.VALIDATION_FAILED
+        try:
+            response = requests.post(settings.GTX_VALIDATION_URL,
+                                     {'asset_physical_id': self.asset_physical_id, 'shipment_id': self.id})
+            if response.ok:
+                validation_status = GTXValidation.VALIDATED
+            else:
+                LOG.info(f'Invalid response from GTX, returned with status {response.status_code}')
+        except NewConnectionError:
+            LOG.info('Connection to GTX failed')
+
+        self.set_gtx_validation(validation_status)
+
+    def set_gtx_validation(self, validation):
+        self.gtx_validation = validation
+        self.gtx_validation_timestamp = datetime.now()
```

Comment:
Be careful with datetime.now(), we need to be explicit about timezones.

Reply:
Changing it to `datetime.utcnow()`

Reply:
https://stackoverflow.com/a/24666683/845745